pkg/network/linux-bridge.go

```diff
@@ -20,6 +29,32 @@ func changeSafeLinuxBridge(prev, next *opv1alpha1.NetworkAddonsConfigSpec) []err
 	return nil
 }
 
+// In older versions of the operator, we used daemon sets of type 'extensions/v1beta1', later we
+// changed that to 'apps/v1'. Because of this change, we are not able to seamlessly upgrade using
+// only Update methods. Following this we find and delete the old daemonSet if configured
+func CleanUpLinuxBridge(ctx context.Context, client k8sclient.Client, objs []*unstructured.Unstructured) []error {
```

Comment:
It is needed but I think I named it wrong. It should be CleanUpBridgeMarker().
Please confirm.

Reply:
That workaround was there to support upgrade from very old versions we do not support now. Therefore I believe we can drop it. If we move it here, I would require a test coverage for that (and that may be really difficult since we changed the namespaces since that version and you would need to change the whole lifecycle tests).

Let's drop the workaround we have there, if something bad happens, we can fix it. But again, that version is not supported for a long time (upgrades were not supported at all at the time).